apps/hexagon_dma/process_err.cpp

```diff
@@ -27,13 +27,9 @@ int main(int argc, char **argv) {
         data_in[i] = ((uint8_t)rand()) >> 1;
     }
 
-    Halide::Runtime::Buffer<uint8_t> input_validation(data_in, width, height, PLANE);
-    Halide::Runtime::Buffer<uint8_t> input(nullptr, width, height, PLANE);
+    Halide::Runtime::Buffer<uint8_t> input_validation(data_in, width, height, PLANE, PLANE);
```

Comment:
It seems unlikely that both of the last two dimensions should be 'PLANE'.

Reply:
These are just dummy values . The error we are checking is for 4 dimension buffers